presentation-api/receiving-ua/support/idlharness_receiving-ua.html

```diff
@@ -24,14 +27,13 @@
 <script id='idl' type="text/plain">
 partial interface Navigator {
     [SameObject]
```

Comment:
Could you also add the `SecureContext` flag that has just made it to the spec here, on the `Presentation` and `PresentationConnectionList` interfaces, as well as on the events interface?

This can be done in another pull request though (we'll need one to update controlling tests in any case).